components/net/bluetooth_thread.rs

```diff
@@ -115,6 +119,24 @@ fn matches_filter(device: &BluetoothDevice, filter: &BluetoothScanfilter) -> boo
             }
         }
     }
+
+// Step 4.
+// TODO: Implement get_manufacturer_data to device.
```

Comment:
`Implement get_manufacturer_data to device.` -> `Implement get_manufacturer_data in device crate.`